osu.Game/Overlays/Rankings/SpotlightSelector.cs

```diff
@@ -50,7 +51,7 @@ public IEnumerable<APISpotlight> Spotlights
         public SpotlightSelector()
         {
             RelativeSizeAxes = Axes.X;
-            Height = 100;
+            Height = 155;
```

Comment:
I kind of disagree with setting this explicitly of making this `AutoSize`d along Y and just making the measurements below match. This is not an explicitly-set width in css.

Reply:
This has to be explicitly set because autosizing causes the dropdown to change the height of the whole selector area when expanded (not to mention other layout issues):

![2020-03-03_22-27-41](https://user-images.githubusercontent.com/27722894/75821279-401bc880-5d9e-11ea-89bc-44a227819710.gif)